pkg/rewards/claim.go

```diff
@@ -15,11 +16,14 @@ import (
 	"github.com/Layr-Labs/eigenlayer-cli/pkg/internal/common/flags"
 	"github.com/Layr-Labs/eigenlayer-cli/pkg/telemetry"
 	"github.com/Layr-Labs/eigenlayer-cli/pkg/utils"
+	"github.com/wealdtech/go-merkletree/v2"
+	"gopkg.in/yaml.v2"
```

Comment:
nit: surprised this isn't alphabetized automatically by go.

Reply:
default formatter doesn't seem to enforce / fix this, will leave as is and we can look into tooling to do this automatically